codec-dns/src/main/java/io/netty/handler/codec/dns/DefaultDnsRecordDecoder.java

```diff
@@ -20,6 +20,9 @@
 import io.netty.util.CharsetUtil;
 import io.netty.util.internal.UnstableApi;
 
+import java.nio.charset.StandardCharsets;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.nio.charset.StandardCharsets'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)